src/Ledger/Ratify.lagda

```diff
@@ -204,7 +204,7 @@ mostStakeDRepDist-∅ {dist} = suc (Σᵐᵛ[ x ← dist ᶠᵐ ] x) , Propertie
 
 ∃topNDRepDist : ∀ {n dist} → lengthˢ (dist ˢ) ≥ n → n > 0
                 → ∃[ c ] lengthˢ (mostStakeDRepDist dist c ˢ) ≥ n
-                       × lengthˢ (mostStakeDRepDist dist (suc c) ˢ) < n
+                       × lengthˢ (mostStakeDRepDist dist (suc c) ˢ) Data.Nat.< n
```

Comment:
Since we don't have a typeclass for `_<_` yet and I wanted to have a `_<_` in the PDF (which compares epochs) I had to change this to be fully qualified (and there's a hacky `_<_` for epochs later in this file).

Reply:
Wouldn't it be better to add hidden `where` clauses that import/define the appropriate for each of the two uses (`Nat` and `Epoch`)? (until the "orders" PR #34 gets merged, that is), e.g. 
```agda
\begin{code}
∃topNDRepDist : ∀ {n dist} → lengthˢ (dist ˢ) ≥ n → n > 0
                → ∃[ c ] lengthˢ (mostStakeDRepDist dist c ˢ) ≥ n
                       × lengthˢ (mostStakeDRepDist dist (suc c) ˢ) < n
                       × lengthˢ (mostStakeDRepDist dist (suc c) ˢ) < n
\end{code}\begin{code}[hide,inline]  where open import Data.Nat using (_<_) \end{code}
...
\begin{code}
expired : Epoch → GovActionState → Set
expired current record { expiresIn = expiresIn } = expiresIn < current
\end{code}\begin{code}[hide,inline] 
  where
    infix 4 _<_
     _<_ : Epoch → Epoch → Set
    a < b = a ≤ b × a ≢ b
\end{code}
```

Reply:
I've put the second one in a `where` block, but I don't really care about the first, since it'll be most likely be removed entirely anyway.